components/slider/index.jsx

```diff
@@ -24,81 +24,91 @@ import { SLIDER } from '../../utilities/constants';
 	 */
-	id: PropTypes.string,
+	'aria-describedby': PropTypes.string,
 	/**
-	 * Name of the submitted form parameter.
+	 * Assistive text for accessibility**
+	 * `label`: Visually hidden label but read out loud by screen readers.
+	 *
 	 */
-	name: PropTypes.string,
+	assistiveText: PropTypes.shape({
+		label: PropTypes.string,
+	}),
 	/**
-	 * This label appears above the Slider.
+	 * Class names to be added to the outer container of the Slider.
 	 */
-	label: PropTypes.string,
+	classNameContainer: PropTypes.oneOfType([
+		PropTypes.array,
+		PropTypes.object,
+		PropTypes.string,
+	]),
 	/**
-	 * The Slider is a controlled component, and will always display this value.
+	 * This is the initial value of an uncontrolled form element and is present only to provide compatibility
+	 * with hybrid framework applications that are not entirely React. It should only be used in an application
+	 * without centralized state (Redux, Flux). "Controlled components" with centralized state is highly recommended.
+	 * See [Code Overview](https://github.com/salesforce/design-system-react/blob/master/docs/codebase-overview.md#controlled-and-uncontrolled-components) for more information.
 	 */
-	value: PropTypes.number,
+	defaultValue: PropTypes.number,
 	/**
-	 * Minimum value of a specified range.
+	 * Disables the Slider and prevents clicking it. Only available on the horizontal view.
 	 */
-	min: PropTypes.number,
+	disabled: PropTypes.bool,
 	/**
-	 * Maximum value of a specified range.
+	 * Message to display when the Slider is in an error state. When this is present, also visually highlights the component as in error.
 	 */
-	max: PropTypes.number,
+	errorText: PropTypes.string,
 	/**
-	 * Indicates the granularity that is expected by limiting the allowed values.
+	 * Set the HTML `id` of the slider.
 	 */
-	step: PropTypes.number,
+	id: PropTypes.string,
 	/**
-	 * Disables the Slider and prevents clicking it.
+	 * This label appears above the Slider.
 	 */
-	disabled: PropTypes.bool,
+	label: PropTypes.string,
 	/**
-	 * Size of the slider.
+	 * Maximum value of a specified range. Defaults to 100.
 	 */
-	size: PropTypes.oneOf(['x-small', 'small', 'medium', 'large']),
+	max: PropTypes.number,
 	/**
-	 * Modifier that makes the slider vertical
+	 * Minimum value of a specified range. Defaults to 0.
 	 */
-	vertical: PropTypes.bool,
+	min: PropTypes.number,
 	/**
-	 * Class names to be added to the outer container of the Slider.
+	 * Name of the submitted form parameter.
 	 */
-	className: PropTypes.oneOfType([
-		PropTypes.array,
-		PropTypes.object,
-		PropTypes.string,
-	]),
+	name: PropTypes.string,
 	/**
-	 * Message to display when the Slider is in an error state. When this is present, also visually highlights the component as in error.
+	 * This event fires whenever the user has modified the data of the control.
```

Comment:
```
* This event fires whenever the user has modified the data of the control. `event, { value: [string] }` is passed in.
```

See https://github.com/salesforce/design-system-react/blob/master/components/combobox/combobox.jsx#L427

Reply:
Oh, sorry. That's exactly what you did. The [tests](https://github.com/salesforce/design-system-react/pull/1317/commits/9529d9afc2d60002321fabf02f910b5ab7eae71b#diff-180f22e91382498f7fbe067108b3a63eR187) though seem to not get an object, but a straight number. The test probably just needs updating.

Reply:
@interactivellama can you clarify which prop you're referring to?

Reply:
Updated to follow the combobox implementation, passing an object with key of value.